knowledge-base/kb-multicolumncombobox-howto-remove-selection-box.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Remove the selection boxes inside RadMultiColumnComboBox
```

Comment:
Use capital letters for the title: Remove the Selection Boxes Inside RadMultiColumnComboBox